zcash_client_backend/proto/service.proto

```diff
@@ -177,6 +197,10 @@ service CompactTxStreamer {
     rpc GetTreeState(BlockID) returns (TreeState) {}
     rpc GetLatestTreeState(Empty) returns (TreeState) {}
 
+    // Returns a stream of information about roots of subtrees of the Sapling and Orchard
+    // note commitment trees.
```

Comment:
```suggestion
    // Returns a stream of information about roots of subtrees of the note commitment tree
    // for the specified shielded protocol (Sapling or Orchard).
```

Reply:
This suggestion needs to be made on the `zcash/lightwalletd` repository, which is the canonical source of this protobuf file.

Reply:
Done: https://github.com/zcash/lightwalletd/issues/446